glide.yaml

```diff
@@ -17,6 +17,8 @@ import:
   version: ^0.8.1
 - package: go.uber.org/atomic
   version: ^1
+- package: github.com/pkg/errors
```

Comment:
can we move this out to a separate change, not really related to mutexes

Reply:
I think this is a fair tradeoff for not implementing (however temporary) ad hoc error wrapping, e.g.:

```go
err = fmt.Errorf("something bad happened: %v", err)
```

is not conventionally used so much as:

```go
err = errors.WithMessage(err, "something bad happened")
```

(same with `errors.Wrap`, etc)

Reply:
Agree, but I think that whole change (changing the error message) should be done separately since it isn't really related to the mutex simplification.